src/plugins/plan/util.js

```diff
@@ -45,6 +45,10 @@ export function getValidatedData(domainObject) {
           groupActivity.end = activity[sourceMap.end];
         }
 
+        if (sourceMap.id) {
+          groupActivity.id = activity[sourceMap.id];
```

Comment:
Is the schema for this defined anywhere in documentation? If we're exposing it to the Open Source we ought to let folks know what a Plan looks like. Otherwise, there's a strong case for a PlanProvider, but that's out of scope for this PR

Reply:
I'll document it.